examples/morpheusvm/cmd/morpheus-cli/cmd/resolutions.go

```diff
@@ -57,26 +57,34 @@ func sendAndWait(
 }
 
 func handleTx(tx *chain.Transaction, result *chain.Result) {
-	summaryStr := string(result.Output)
-	actor := tx.Auth.Actor()
 	status := "❌"
 	if result.Success {
 		status = "✅"
-		switch action := tx.Action.(type) { //nolint:gocritic
-		case *actions.Transfer:
-			summaryStr = fmt.Sprintf("%s %s -> %s", utils.FormatBalance(action.Value, consts.Decimals), consts.Symbol, codec.MustAddressBech32(consts.HRP, action.To))
+	}
+	for i := 0; i < len(result.Outputs); i++ {
+		for j := 0; j < len(result.Outputs[i]); j++ {
+			actor := tx.Auth.Actor()
+			for _, action := range tx.Actions {
+				summaryStr := string(result.Outputs[i][j])
+				if result.Success {
```

Comment:
You can be smarter about this I think....if the tx is not successful, there is only 1 error (according to your earlier handling).